widgetssdk/src/main/java/com/glia/widgets/core/queue/GliaQueueRepository.kt

```diff
@@ -2,9 +2,38 @@ package com.glia.widgets.core.queue
 
 import com.glia.androidsdk.queuing.Queue
 import com.glia.widgets.di.GliaCore
+import com.glia.widgets.helper.Logger
+import com.glia.widgets.helper.TAG
+import io.reactivex.rxjava3.core.Flowable
 import io.reactivex.rxjava3.core.Single
+import io.reactivex.rxjava3.processors.BehaviorProcessor
+
+internal sealed interface FetchQueuesState {
+    object Loading : FetchQueuesState
+    data class Queues(val queues: List<Queue>) : FetchQueuesState
+    object Error : FetchQueuesState
+}
 
 internal class GliaQueueRepository(private val gliaCore: GliaCore) {
+
+    private val _queues: BehaviorProcessor<FetchQueuesState> = BehaviorProcessor.createDefault(FetchQueuesState.Loading)
```

Comment:
Since I have been seeing this often lately, I wanted to confirm for consistency: Do we have some specific time for when to use underscore for property naming and when not to?